scalyr_agent/builtin_monitors/windows_event_log_monitor.py

```diff
@@ -814,6 +816,19 @@ def run(self):
         if isinstance(self.__api, OldApi):
             self._logger.info("Evt API not detected.  Using older EventLog API")
 
+            maximum_records = self._config.get("maximum_records_per_source")
+            if (
+                maximum_records is not None
+                and maximum_records != DEFAULT_MAXIMUM_RECORDS_PER_SOURCE
+            ):
+                # This option is only applicable when using old API so we emit a warning is this
+                # config option is set to a non default value when using a new API
+                self._logger.warn(
```

Comment:
Maybe this message should be shown with a new API then?

Reply:
Thanks, good catch.

That was an error on my side -  I put it in a wrong branch. I wanted to add unit tests, but it's a bit of PITA.